snooty/test_product_landing.py

```diff
@@ -0,0 +1,171 @@
+from pathlib import Path
+
+import pytest
+
+from . import n
+from .parser import Project
+from .test_project import Backend
+from .types import BuildIdentifierSet, FileId
+from .util_test import check_ast_testing_string
+
+
+@pytest.fixture
+def backend() -> Backend:
+    backend = Backend()
+    build_identifiers: BuildIdentifierSet = {}
+    with Project(
+        Path("test_data/test_product_landing"), backend, build_identifiers
+    ) as project:
+        project.build()
+
+    return backend
+
+
+def test_queryable_fields(backend: Backend) -> None:
```

Comment:
This isn't really about testing queryable fields, in devhub we extract certain directives to be queried by the front end but we're not doing anything beyond the standard here.